packages/contracts-bedrock/contracts/L1/ResourceMetering.sol

```diff
@@ -11,7 +11,7 @@ import { Burn } from "../libraries/Burn.sol";
  * @notice ResourceMetering implements an EIP-1559 style resource metering system where pricing
  *         updates automatically based on current demand.
  */
-contract ResourceMetering {
+abstract contract ResourceMetering {
```

Comment:
Why is this marked abstract? It doesn't have any un-implemented functions, not sure if it needs this. Non-blocking though.

Reply:
Yeah, good call out... 
I think I made it `abstract` because of long-ish chain of things that are no longer entirely true.
So, I'll add a constructor and remove `abstract`.

Reply:
Did this [here](https://github.com/ethereum-optimism/optimism/pull/2825/commits/84868c79dff3a3385f509b8bf192991fd1947cf2).